store/details.go

```diff
@@ -55,3 +58,22 @@ type snapDetails struct {
 	Private     bool   `json:"private"`
 	Confinement string `json:"confinement"`
 }
+
+type snapDeltaDetail struct {
+	FromRevision    int    `json:"from_revision"`
+	ToRevision      int    `json:"to_revision"`
+	Format          string `json:"format"`
+	AnonDownloadURL string `json:"anon_download_url,omitempty"`
+	DownloadURL     string `json:"download_url,omitempty"`
+	Size            int64  `json:"binary_filesize,omitempty"`
+	Sha3_384        string `json:"download_sha3_384,omitempty"`
+}
+
+func getDefaultDetailFields() []string {
+	fields := getStructFields(snapDetails{})
+	// Don't include deltas in the default DetailFields. Instead they'll
+	// be requested explicitly in ListRefresh only.
+	i := sort.SearchStrings(fields, "deltas")
```

Comment:
Yes, let's please not do that. Clarifying @cihpaca's comment, a sorted list of fields is less clear than an unsorted one, because a struct is put in place with semantic proximity, which is much nicer than alphabetical one in this case. Let's please avoid both this and the prior point at once by not manipulating that list based on whether we want deltas or not.